src/cadence/scripts/getNFTIDs.cdc

```diff
@@ -152,5 +152,26 @@ pub fun main(ownerAddress: Address): {String: [UInt64]} {
             ids["Gooberz"] = col.getIDs()
         }
 
+    if let col = owner.getCapability(MintStoreItem.CollectionPublicPath)
+        .borrow<&{MintStoreItem.MintStoreItemCollectionPublic}>() {
+            ids["MintStoreItem"] = col.getIDs()
+        }
+        
+    if let col = owner.getCapability(MintStoreItem.CollectionPublicPath)
```

Comment:
The intended use-case for this script is to retrieve NFT IDs owned by a particular account address. The merchant based filtering is out of scope for this script, but we can keep this in mind as we build out more infrastructure for the NFT API. Can we remove the merchant based ID filtering?

Reply:
Understood. 

Right now we have "MintStoreItem" which aggregates all NFTs, and merchant specific entries "MintStoreItem.<merchantName>".

Our merchants often prefer to be disassociated from each other in user interfaces, since they have nothing in common with each other other.

Instead of having both "MintStoreItem" and "MintStoreItem.<merchant>" lists, can we only keep the "MintStoreItem.<merchant>" lists instead?

Reply:
Our NFT API retrieves the NFTs owned by an account - so if I understand this correctly, if an owner owns NFTs from multiple merchants, it should be okay to display these together. So we should keep the first part of the code (ids["MintStoreItem"])

Reply:
Hey Matthew, thank you for the clarification - we have some validation logic in our nft api for contract address/name filters (eg retrieve all NFTs for an owner belonging to "x" contract) - curious if there are any prefixes here that we can use for request validation?

Would you also be able to update the new test files and add testnet support (optional)?